src/audits/DataTableHeadersMissing.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.browserUtils');
+goog.require('axs.constants.Severity');
+
+(function () {
+    function tableDoesNotHaveHeaderRow(rows) {
```

Comment:
Would you be able to move these three methods into AccessibilityUtils as something like 
```
/**
 * @param {HTMLTableElement} table
 * @return {Array<HTMLTableRowElement>}
 */
axs.utils.headerRowsForTable = function(table) {
    // ...
};

/**
 * @param {HTMLTableElement} table
 * @return {Array<HTMLTableColumnElement>}
 */
axs.utils.headerColumnsForTable = function(table) {
    // ...
};

/**
 * @param {HTMLTableElement} table
 * @return {boolean}
 */
axs.utils.isDataTable = function(table) {
    //...
};